Lickability/PinpointKit
# Adds Unit Tests For SystemLogCollector

## What it Does

Adds unit tests for the `SystemLogCollector`.
Adds extension on XCTestCase for easing the creation of expectations.
Improves ability of `SystemLogCollector` to be used as an instance.

## How to Test

Run the unit tests, make sure they pass.
Open up the sample app and ensure the log that is collected there still displays.

## Notes

There is a `dispatch_after` and expectations in the unit tests to ensure that the asynchronous writing of the logs have finished by the time the test attempts to retrieve them. This prevents occasionally failing tests. Terrible, I know.
